Update deprecation.md (#4183)
SumoLogic/sumologic-documentation: docs/alerts/scheduled-searches/deprecation.md

## What do I need to do?

> Before **May 15, 2025**, please migrate any Real-Time Scheduled Searches to either Monitors or reduce its frequency to the minimum of 15m or another suitable timerange. Any Real-Time Scheduled Searches that remain after the deprecation date will automatically be converted to 15m schedules. For each automatic conversion, there will be a corresponding audit log for this activity written to your Sumo instance. 

Reviewer: ```suggestion
Before **May 15, 2025**, please migrate any Real-Time Scheduled Searches to either Monitors or reduce their frequency to the minimum of 15m or another suitable time range. Any Real-Time Scheduled Searches that remain after the deprecation date will automatically be converted to 15m schedules. For each automatic conversion, there will be a corresponding audit log for this activity written to your Sumo Logic instance. 
```

Author: @jpipkin1 Yes, good on all changes, thanks for that.
